arbitrum-docs/partials/_contribute-docs-partial.md

```diff
@@ -0,0 +1,258 @@
+    - Ensure that your document's YAML frontmatter contains a `third_party_content_owner` property, with the Github username of the designated maintainer. This person will be assigned to your document's issues and PRs, and will be expected to resolve them in a timely manner.
+ 4. **Organization**
+    - Third-party docs are organized within the `Third-party content` node located at the bottom of each documentation section’s sidebar.
+    - This node’s content is grouped by third-party product. If/when this becomes unwieldy, we'll begin grouping products by [portal](https://portal.arbitrum.io/) category.
+ 5. **Limited document types**
+    - To manage our team's limited capacity, third-party documents must be either ***Quickstarts***, ***How-tos***, or ***Concepts***. See [document types](#document-type-conventions) below for guidance.
+ 6. **Incremental contributions: One document at a time, procedures first**
+    - Third-party document PRs should contain at most one new document.
+    - Any given product's first docs contribution should be a ***Quickstart*** or ***How-to***.
+    - Additional documents will be merged only if we can verify that our readers are deriving value from your initial contribution.
+    - The way that we verify this isn't yet formally established, and it isn't publicly disclosed. Our current approach combines a number of objective and subjective measures.
+ 7. **Policy acknowledgment**
+    - Before merging third-party documentation PRs, we ask contributors to acknowledge that they've read, understood, and agree with the following policies: 
+      1. **Content ownership**: As the author, you retain ownership of and responsibility for the content you contribute. You're free to use your content in any way you see fit outside of Arbitrum's docs. Remember that when contributing content to our documentation, you must ensure you have the necessary rights to do so, and that the content doesn't infringe on the intellectual property rights of others.
+      2. **License for use**: By contributing your content to our documentation, you grant Offchain Labs a non-exclusive, royalty-free license to use, reproduce, adapt, translate, distribute, and display the content in our documentation. This allows us to integrate your content into our docs and make it available to all users.
+      3. **Right to modify or remove**: Offchain Labs reserves the right to modify or remove third-party content from our documentation at any time. This might be necessary due to a range of reasons, such as content becoming outdated, receiving very low pageviews over an extended period, or misalignment with our guidelines or goals.
+
+## Content conventions
+
+Offchain Labs supports [Arbitrum docs](/) and [Prysm docs](https://docs.prylabs.network/docs/getting-started). The following **content conventions** help us maintain consistency across these documentation sets:
```

Comment:
Should we add the governance docs here too? (Not sure)

Reply:
Negative - this was removed in the most recent edit in response to internal feedback. We'd have to block this on legal review if we explicitly associate this process/policy with the DAO docs.